community/ibm-crate-dev/README.md

```diff
@@ -0,0 +1,90 @@
+# CrateDB
+
+[CrateDB](https://crate.io/)A distributed SQL DBMS built atop NoSQL storage & indexing delivers the best of SQL & NoSQL in one database. 
+
+```console
+$ helm install stable/ibm-crate-dev
+```
+
+## Prerequisites
+
+- Kubernetes 1.7+ 
+- Tiller 2.7.2 or  later
+
+## Resources Required
+The chart deploys pods consuming minimum resources as specified in the resources configuration parameter (default: Memory: 200Mi, CPU: 100m)
+
+
+## PodSecurityPolicy Requirements
+This chart requires a PodSecurityPolicy to be bound to the target namespace prior to installation. Choose predefined ibm-anyuid-psp PodSecurityPolicy.
+
+## Introduction
+
+This chart bootstraps a [CrateDB](https://github.com/crate/crate) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+$ helm intall --name my-release stable/ibm-crate-dev
+```
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```console
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Chart Details
+This chart bootstraps a [CrateDB](https://hub.docker.com/r/ibmcom/crate-ppc64le/) deployment on a [Kubernetes](http://kubernetes.io) cluster
+
+
+## Configuration
+
+The following table lists the configurable parameters of the CrateDB chart and their default values.
+
+|      Parameter            |          Description            |                         Default                         |
+|---------------------------|---------------------------------|---------------------------------------------------------|
+| `image`                   | The image to pull and run       | ibmcom/crate-ppc64le:1.0.5                              |
```

Comment:
Is this really crate v 1.0.5?  The image is over a year old and crate has now two major versions ahead (3.2.3)
https://crate.io/docs/crate/reference/en/latest/appendices/release-notes/3.2.3.html